### ISSUE TITLE
About the adding SAD to the model

### ISSUE THREAD

**Issue by ixtiyoruz** [cardwing/Codes-for-Lane-Detection#181]
Hi, thank you for the great work. But i have one question, so here is the Enet model:
![image](https://user-images.githubusercontent.com/24115387/68179484-663d8180-ffd3-11e9-9212-58906627270a.png)

In the paper you said you have chosen middle and later layer to add SAD, so i cant use output of first section right ? . Did you change the model structure ?
I am trying to reimplement it in Tensorflow.

**Comment by cardwing** [comment]
Yes. And I have changed the original network architecture of ENet. More details can be found in ENet-Label-Torch and my paper.

**Comment by ixtiyoruz** [comment]
Thank you for the quick response, as far i understand you have added two more section 2. I haven't applied the SAD yet but it is predicting road labels very well, after training 10k samples ))
